client/state/http/test/index.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { HTTP_REQUEST } from 'state/action-types';
+import useNock, { nock } from 'test/helpers/use-nock';
+import { extendAction } from 'state/utils';
+import { failureMeta, successMeta } from 'state/data-layer/wpcom-http';
+import actionHandlers from '../';
+
+const httpHandler = actionHandlers[ HTTP_REQUEST ][ 0 ];
```

Comment:
this is really confusing here through the indirection. would there be a problem using this instead?

```js
import { httpHandler } from '../';
```